mobius-core/src/main/java/com/spotify/mobius/EventProcessor.java

```diff
@@ -22,8 +22,6 @@
 import static com.spotify.mobius.internal_util.Preconditions.checkNotNull;
 
 import com.spotify.mobius.functions.Consumer;
-import java.util.ArrayList;
```

Comment:
Feels like we can inline this in the loop now, no? I don't know if it's really providing much value but maybe that's a separate change.

Reply:
Yes I was thinking that would be a separate PR